### PR TITLE
Update LN Markets to v1.2.3

### PR DESCRIPTION
**Pull request by vafanassieff**
Hey new update to LNM app that should fix login issue and with a new auth :)

BTW, where can I add our new logo ?

Thanks !

### CODE REVIEW

**Comment by louneskmt** [comment]
Hey, thanks for this PR! Regarding the new logo, you can make a PR to https://github.com/getumbrel/umbrel-apps-gallery.

File: apps/lnmarkets/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   lnmarkets:
-    image: ghcr.io/ln-markets/umbrel:v1.1.5@sha256:94b2259e4c2445d518cabc4bd6c0519c5fae87a3fb3c0bb564522e805d63bd89
+    image: ghcr.io/ln-markets/umbrel:v1.2.3@sha256:2070b9d8e57f6eb997919e0f52b90033c7492870f0b54c88c590860a67df4704
```
Comment (nevets963) [inline apps/lnmarkets/docker-compose.yml:5]: Can you change the digest to: `sha256:c11ec6d28d1b1d11d82ae8b33916732686a296beb21c9c3eaee80c823bd2d00f`

This digest looks specific to `arm64` but the above is good for `amd64` and `arm64`. Other than that, this is good to be merged in.

**Review by nevets963 — CHANGES_REQUESTED** [review]

File: apps/lnmarkets/docker-compose.yml (hunk 1 above)
Comment (lukechilds) [inline apps/lnmarkets/docker-compose.yml:5]:
```suggestion
    image: ghcr.io/ln-markets/umbrel:v1.2.3@sha256:c11ec6d28d1b1d11d82ae8b33916732686a296beb21c9c3eaee80c823bd2d00f
```

**Comment by lukechilds** [comment]
Hey guys, resolved the issue in https://github.com/getumbrel/umbrel/pull/1380

@vafanassieff in future are you able to submit PRs from a feature branch instead of master so we're able to push changes to it?

**Comment by vafanassieff** [comment]
@lukechilds Crap, we are always missing something, duly noted !

Thanks for the merge !

**Comment by lukechilds** [comment]
No worries!

**Pull request closed without merging**